app/components/LinksGroup/LinksGroup.tsx

```diff
@@ -0,0 +1,152 @@
+import {
+  Group,
+  Box,
+  Collapse,
+  Text,
+  UnstyledButton,
+  IconCaretRight,
+  useMantineTheme,
+  MantineTheme,
+  List,
+} from "@pokt-foundation/pocket-blocks"
+import { useLocation } from "@remix-run/react"
+import { useState } from "react"
+
+type GetTextColor = (isActive: boolean, theme: MantineTheme, size?: string) => string
+
+const getTextColor: GetTextColor = (isActive, theme, size = "") => {
+  if (isActive) {
+    return theme.colors.blue[5]
+  }
+  return size === "lg" ? "#fff" : theme.colors.navy[0]
+}
+
+type GetFontSize = (nesting_level: number, size?: string) => string
+
+const getFontSize: GetFontSize = (nesting_level, size) => {
+  if (size && size === "lg") {
+    return "18px"
+  }
+  if (nesting_level) {
+    return "15px"
+  }
+  return "16px"
+}
+
+type LinkItemProps = {
+  isActive: boolean
+  label: string
+  link: string
+  nesting_level: number
+  size?: string
+  theme: MantineTheme
+}
+
+const LinkItem = ({
+  isActive,
+  label,
+  link,
+  nesting_level,
+  size,
+  theme,
+}: LinkItemProps) => (
+  <Text
+    m={0}
+    p={nesting_level ? `10.5px 0 10.5px ${nesting_level * 32}px` : "16px 8px"}
+    sx={{
+      color: getTextColor(isActive, theme, size),
+      fontSize: getFontSize(nesting_level, size),
+      fontWeight: size === "lg" ? "bold" : "normal",
+      textTransform: "capitalize",
+    }}
+  >
+    <a href={link}>{label}</a>
```

Comment:
I think we can improve the performance if we use `Link` component from Remix, and use the prefetch prop with "intent". Navigation will be much faster and feel snappier. 

https://remix.run/docs/en/main/components/link#prefetch